app/views/taxons/show.html.erb

```diff
@@ -14,10 +14,9 @@
       locals: {
         message: <<-MESSAGE
         This is a test version of the layout of this page.
-        <a href='https://www.smartsurvey.co.uk/s/betasurvey2017'>
+        <a id=taxonomy-survey href='https://www.smartsurvey.co.uk/s/betasurvey2017' target='_blank'>
```

Comment:
We need to add [`rel='noopener noreferrer'`](https://github.com/alphagov/government-frontend/commit/def3f773697c01dd1ce9153e9c0fcf2961f0fea7)